samples/tab-deeplink-all-features/csharp/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-deeplink-all-features-csharp",
+    "source": "officeDev",
+    "title": "DeepLink All Features",
+    "shortDescription": "Microsoft Teams sample app for demonstrating deeplink all features",
```

Comment:
Microsoft Teams sample app for demonstrating deep links features

Reply:
fixed